best.html

```diff
@@ -12,7 +12,7 @@
 
 <p>Functional testing is difficult to get right for many reasons.
  As if application state, complexity, and dependencies don't make testing difficult enough,
- dealing with browsers – and especially cross-browser incompatibilities –
+ dealing with browsers (especially with cross-browser incompatibilities)
```

Comment:
Keep dashes, alternatively use commas.

The dashes are currently used to highlight that this is an important fact, and placing it in parenthesis makes for the opposite effect.